aws-observability-terraform/terraform-aws-observability-common/alb_source.tf

```diff
@@ -0,0 +1,53 @@
+resource "sumologic_elb_source" "this" {
+  for_each = toset(var.manage_alb_logs_source ? ["this"] : [])
+
+  category      = "aws/observability/alb/logs"
+  collector_id  = sumologic_collector.hosted["this"].id
+  content_type  = "AwsElbBucket"
+  name          = var.alb_logs_source_name
+  paused        = false
+  scan_interval = var.scan_interval
+
+  authentication {
+    type     = "AWSRoleBasedAuthentication"
+    role_arn = aws_iam_role.sumologic_source["this"].id
+  }
+
```

Comment:
We will have to provide cutoff_relative_time=-1d and fields as per CF here, link https://github.com/hashicorp/terraform-provider-sumologic/tree/master/website#common-source-properties